test/deploy/linux/redis/install/debian/roles/configure/tasks/main.yml

```diff
@@ -26,7 +26,7 @@
 
 - block:
   - name: Export PASSWORD
-    shell: "echo export NR_CLI_DB_PASSWORD=Virtuoso4all! >> ~/.bashrc"
+    shell: "echo export NR_CLI_PASSWORD=Virtuoso4all! >> ~/.bashrc"
```

Comment:
@carlossscastro I don't want to bother with the semantic, I'll merge, but it might be good to be consistent with the other recipes since we use NR_CLI_DB_PASSWORD in other recipes. I agree it would be subjective to consider redis as a DB though.